lib/reporters/base.js

```diff
@@ -168,11 +168,17 @@ exports.list = function(failures){
     var err = test.err
       , message = err.message || ''
       , stack = err.stack || message
-      , index = stack.indexOf(message) + message.length
-      , msg = stack.slice(0, index)
+      , index = stack.indexOf(message)
       , actual = err.actual
       , expected = err.expected
       , escape = true;
+    if (index === -1) {
+      msg = message;
+    } else {
+      index += message.length;
+      msg = stack.slice(0, index);
+      stack = stack.slice(index + 1)
```

Comment:
Oh yes, thank you. I removed the `stack.slice` call at line 208, added a missing `;` and fixed the code doc. Please have another look.